tensorboard/webapp/plugins/npmi/metadata.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorboard.compat.proto import summary_pb2
+
+from tensorboard.webapp.plugins.npmi import plugin_data_pb2
+
+PLUGIN_NAME = "npmi"
+
+
+def create_summary_metadata(description, title):
+    content = plugin_data_pb2.NpmiPluginData(title=title)
+    return summary_pb2.SummaryMetadata(
+        summary_description=description,
```

Comment:
Per @wchargin let's add `data_class = summary_pb2.DATA_CLASS_TENSOR` here so that the DataProvider will return this summary data for a `list_tensors()` call

Reply:
Done in [5cee167](https://github.com/tensorflow/tensorboard/pull/3799/commits/5cee16744ea491af7e482381b7f4dc15b2428522)